lms/services/__init__.py

```diff
@@ -1,6 +1,5 @@
 from lms.services.canvas import CanvasService
 from lms.services.exceptions import (
-    BlackboardFileNotFoundInCourse,
```

Comment:
It's now the view rather than the service that raises `BlackboardFileNotFoundInCourse` so the exception class itself has moved into the views.